vendor.conf

```diff
@@ -14,8 +14,8 @@ github.com/coreos/go-systemd v17
 github.com/coreos/pkg fa29b1d70f0beaddd4c7021607cc3c3be8ce94b8
 github.com/deckarep/golang-set ef32fa3046d9f249d399f98ebaf9be944430fd1d
 
-github.com/docker/docker 162ba6016def672690ee4a1f3978368853a1e149
 github.com/docker/go-connections 7beb39f0b969b075d1325fecb092faf27fd357b6
+github.com/docker/docker 71cd53e4a197b303c6ba086bd584ffd67a884281
```

Comment:
this PR was a bit older, and looks like this reverted moby to an older commit; https://github.com/moby/moby/compare/162ba6016def672690ee4a1f3978368853a1e149...71cd53e4a197b303c6ba086bd584ffd67a884281

these are the changes that were reverted; https://github.com/moby/moby/compare/71cd53e4a197b303c6ba086bd584ffd67a884281...162ba6016def672690ee4a1f3978368853a1e149